src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const liElements = document.querySelectorAll('li');
+
+liElements.forEach((item) => {
+  if (item.childNodes.length > 0) {
+    const textNode = item.childNodes[0];
+    const textContent = textNode.textContent;
```

Comment:
The code assumes that the first child node is always a text node. This might not be true if there are other elements or text nodes present. Consider checking if `textNode.nodeType === Node.TEXT_NODE` before accessing `textContent` to ensure it's a text node.